src/assets/locales/fr.json

```diff
@@ -0,0 +1,465 @@
+  "autoexit.idle_subtitle": "Vous avez été inactif trop longtemps.",
+  "autoexit.cancel": "ANNULER",
+  "presence.entered_room": "est entré dans la salle.",
+  "presence.entered_lobby": "est arrivé dans le hall.",
+  "presence.join_lobby": "a rejoint le hall.",
+  "presence.join_room": "a rejoint la salle.",
+  "presence.leave": "est parti.",
+  "presence.name_change": "s'appelle à présent",
+  "presence.scene_change": "a choisi la scène",
+  "presence.hub_name_change": "a renommé la salle",
+  "presence.in_lobby": "Hall",
+  "presence.entering": "Arrive dans la salle",
+  "presence.in_room": "Dans la salle",
+  "presence.mute_all": "Couper le son de tous les participants",
+  "home.create_a_room": "Créer une salle",
+  "home.desktop.add_pwa": "Installer une app Desktop",
+  "home.mobile.add_pwa": "Ajouter à l'écran d'accueil",
+  "home.take_a_tour": "Faire une visite",
+  "home.room_create_options": "options",
+  "home.room_create_button": "Créer une salle",
+  "home.create_name.validation_warning": "Nom invalide, limité entre 4 et 64 caractères, symboles restreints.",
+  "home.powered_by_prefix": "Propulsé par",
+  "home.powered_by_link": "Hubs Cloud",
+  "home.join_us": "Rejoignez la conversation",
+  "home.subscribe_to_mailing_list": "Abonnez-vous pour recevoir les dernières nouvelles",
+  "home.have_code": "Vous avez le code d'une salle ?",
+  "home.add_to_discord_1": "Ajouter le",
+  "home.add_to_discord_2": "%app-name% Bot",
+  "home.add_to_discord_3": "à Discord",
+  "home.create_with_spoke": "Créer une scène",
+  "home.report_issue": "Signaler les problèmes",
+  "home.source_link": "Source",
+  "home.whats_new_link": "Actualités",
+  "home.about_link": "A propos",
+  "home.community_link": "Communauté",
+  "home.docs_link": "Documentation",
+  "home.cloud_link": "Hubs Cloud",
+  "home.admin": "Administration",
+  "home.privacy_notice": "Politique de confidentialité",
+  "home.terms_of_use": "Conditions d'utilisation",
+  "home.made_with_love": "fait avec 🦆 par ",
+  "home.environment_author_by": " par ",
+  "home.rooms_title": "Créez des salles en un clic",
+  "home.rooms_blurb": "Partagez des espaces virtuels avec vos amis, vos collègues et vos communautés. Lorsque vous créez une salle avec Hubs, vous pouvez partager sans délai un lien d'invitation vers un espace virtuel privé - aucun téléchargement requis, ni casque de réalité virtuelle.",
+  "home.communicate_title": "Communiquez de manière spontanée",
+  "home.communicate_blurb": "Choisissez un avatar qui vous représente, mettez vos écouteurs et plongez ! Avec Hubs, c'est facile de rester connecté avec d'autres personnes dans une salle privée, en discutant et en chattant.",
+  "home.media_title": "La manière la plus simple de partager un média",
+  "home.media_blurb": "Partagez des contenus avec les autres participants de votre salle en glissant / déposant des photos, des vidéos, des fichiers PDF, des liens et des modèles 3D, directement dans votre espace virtuel.",
```

Comment:
```suggestion
  "home.media_blurb": "Partager du contenu avec les autres participants de la salle en glissant / déposant des photos, des vidéos, des fichiers PDF, des liens ou des modèles 3D, directement dans votre espace virtuel.",
```

Reply:
Not sure if it is "Choose" or "To choose" in the original version ?

Reply:
You mean `Share` ?

https://github.com/mozilla/hubs/blob/f2d17fc3d745c60f7cada1a9b236a806f635c956/src/assets/locales/en.json#L249

I think the infinitive form is better in our situation, we do not want to force something to the user.
The imperative version `Partagez` is more prescriptive than the infinitive version.

Reply:
Oh yes "Share", not "Choose"

The infinitive version sounds a bit weird in this case. It is used when we interact with a machine, for instance, or a tutorial, but here it is more like showing the functionalities of the tool. To me, the user doesn't feel "forced" with the imperative form. 

If you prefer the infinitive version though, let's check that all the similar texts have been translated that way: all the "home" list should be infinitive?

Reply:
Yes I agree with you "Partagez des contenus" is more convenient.

We must write infinitive for action, like button I think, for instance: `home.create_a_room` and in other cases we must give information as you said `it is more like showing the functionalities of the tool.`

Is it ok ?

Reply:
I agree